src/Numerics.jl

```diff
@@ -33,8 +34,11 @@ Base.:*(a::Tensor, b::Tensor) = contract(a, b)
 Base.:*(a::Tensor, b) = contract(a, b)
 Base.:*(a, b::Tensor) = contract(a, b)
 
-function LinearAlgebra.svd(t::Tensor; left_inds=(), kwargs...)
-    if any(∉(labels(t)), left_inds)
+function LinearAlgebra.svd(t::Tensor, left_inds=(); kwargs...)
```

Comment:
I tried to put it like this in the beginning, but the problem is that then I cannot do something like:
```julia
svd(tensor, left_inds=(:l,))
```

And I prefer to put `left_inds` explicitly because it is readable, unlike the other option.

Maybe we could solve it with two methods:
```suggestion
LinearAlgebra.svd(t::Tensor; left_inds=(), kwargs...) = svd(tensor, left_inds; kwargs...)

function LinearAlgebra.svd(t::Tensor, left_inds; kwargs...)
```

Reply:
Okay, this is a good idea.